src/components/material/material-buttons/helper.ts

```diff
@@ -19,13 +18,19 @@ export const hasCorrectAccessType = (
   return manifest.accessTypes.some((type) => type.code === desiredAccessType);
 };
 
-export const isArticle = (manifestation: Manifestation) => {
-  const allMaterialTypes = manifestation.materialTypes.map((materialType) =>
-    materialType.specific.toLowerCase()
+export const hasCorrectMaterialType = (
+  desiredMaterialType: string,
+  manifestation: Manifestation
+) => {
+  return manifestation.materialTypes.some(
+    (type) => type.specific.toLowerCase() === desiredMaterialType.toLowerCase()
   );
-  return (
-    intersection(allMaterialTypes, ["tidsskriftsartikel", "avisartikel"])
-      .length > 0
+};
+
+export const isArticle = (manifestation: Manifestation) => {
+  return !!(
+    hasCorrectMaterialType("tidsskriftsartikel", manifestation) ||
+    hasCorrectMaterialType("avisartikel", manifestation)
   );
```

Comment:
Why do we need double negation/`!` here?


The following returns a boolean: 

```
(
    hasCorrectMaterialType("tidsskriftsartikel", manifestation) ||
    hasCorrectMaterialType("avisartikel", manifestation)
)
```  

Double-negation results in the original value. We should even require any type juggling.